### PR TITLE
Deb updates

### PR DESCRIPTION
**Pull request by errordeveloper**
@mikedanese @luxas PTAL

### CODE REVIEW

File: debian/build.py [hunk 1]
```diff
@@ -46,11 +46,11 @@ def main():
   packages = {
     'kubectl': (
       ('1.3.7', '00'),
-      ('1.4.0-beta.3', '02'),
+      ('1.4.0-beta.5', '02'),
```
Comment (mikedanese) [inline debian/build.py]: Revision can go back to 00

File: debian/xenial/kubernetes-cni/debian/.gitignore [hunk 2]
```diff
@@ -1,3 +1,2 @@
 k8s-cni
 files
```
Comment (mikedanese) [inline debian/xenial/kubernetes-cni/debian/.gitignore]: Can you delete this file?

**Comment by luxas** [comment]
No more specific comments from me apart from fixing the two nits @mikedanese commented

File: debian/build.py (hunk 1 above)
Comment (errordeveloper) [inline debian/build.py]: Didn't know what that number was... thanks.

File: debian/xenial/kubernetes-cni/debian/.gitignore (hunk 2 above)
Comment (errordeveloper) [inline debian/xenial/kubernetes-cni/debian/.gitignore]: Sure, deleted it now.

**Comment by errordeveloper** [comment]
Ok, should be good now.

**Comment by luxas** [comment]
LGTM

File: debian/build.py (hunk 1 above)
Comment (mikedanese) [inline debian/build.py]: We can use it to release multiple versions of the same release in case of crisis. It's an escape hatch.

Comment (errordeveloper) [inline debian/build.py]: Sure thing. Just didn't realise this was the case. May be we could convert this list to a dictionary, so it's more descriptive.

**Pull request merged**